source/index.html.md.erb

```diff
@@ -9407,7 +9443,13 @@ Searches all Sheets that the User can access, for the specified text.
 -----|-----|
 <%= SCOPE %> | READ_SHEETS
 <%= HEADERS %> | <%= HEADER_AUTH %>
-<%= PARAMETERS %> | **query** (required): Text with which to perform the search. Enclose in double-quotes for an exact search.
+<%= PARAMETERS %> | **query** (required): text with which to perform the search. Enclose in double-quotes for an exact search.
+ | **include** (optional): when specified with a value of "favoriteFlag", response includes:<ul><li>favorite -- Folders, Reports, Sheets, Sights, Templates, and Workspaces will have the property "favorite": true</li><li>parentObjectFavorite -- Attachments, Discussions, Profile Fields, and Rows will have the property "parentObjectFavorite": true</li></ul>
+ | **level** (optional): when specified with one of the following values, temporarily disables filters:<ul><li>0 -- allows all filters except for Dashboard, Folder, or Workspace</li><li>1 -- allows all filters except for Folder or Workspace</li><li>2 -- allows all filters</li></ul>
```

Comment:
level flags are internal (undocumented) for mobile. Please remove

Reply:
Done.